synapse/federation/federation_client.py

```diff
@@ -524,7 +525,17 @@ async def _try_destination_list(
                     e.code,
                     e.args[0],
                 )
+            except ProxiedRequestError as e:
+                # If an auth error comes back from a federation endpoint, pass
+                # it back to the client.
+                if e.code == 403:
+                    raise
+
+                logger.warning(
+                    "Failed to %s via %s", description, destination, exc_info=True
+                )
             except Exception:
+                logger.warning("POMP POMP POMP POMP POMP")
```

Comment:
ETOOMUCHPOMP

Reply:
Oh, oops. 😢 Must have been trying to track some code...